src/config.txt

```diff
@@ -0,0 +1,5 @@
+[credentials]
+CONSUMER_KEY = <CONSUMER_KEY>
+CONSUMER_SECRET = <CONSUMER_SECRET>
+ACCESS_KEY = <ACCESS_KEY>
+ACCESS_SECRET = <ACCESS_SECRET>
```

Comment:
Make sure to remove `config.py` from your commit, because it can be considered as obsolete.